pyiron/vasp/base.py

```diff
@@ -466,27 +466,37 @@ def collect_errors(self):
         """
         Collects errors from the VASP run
         """
-        num_eddrmm, snap = self._get_eddrmm_info()
+        num_eddrmm, snap_eddrmm = self._get_eddrmm_info()
```

Comment:
I noticed that we are reading the entire `error.out` file in this function and also do the same thing later on for the `ZBRENT` error. Wouldn't a better way be to read the `error.out` file once and then pass the lines as the argument to this function? This way we avoid unnecessary io.

Reply:
Sure, I wrote now everything in `collect_errors`. This should also be sufficient for future implementations.